querygen/userjourney_fs.go

```diff
@@ -41,8 +41,8 @@ var UserJourneyFieldSet = FieldSet{
 		srcFields: []string{"responseObject.status.conditions{}.message"},
 	},
 	"status_reason": {
-		subObj:    "properties",
-		srcFields: []string{"responseObject.status.conditions{}.reason"},
+		subObj:  "properties",
+		srcExpr: `mvindex('responseObject.status.conditions{}.reason', 0)`,
```

Comment:
We need to use the same `mvindex` workaround on the `status_message` FieldSet entry too.

Reply:
I got something like and didn't notice it:

```
{
  "apiGroup": "appstudio.redhat.com",
  "apiVersion": "v1alpha1",
  "kind": "releases",
  "name": "fbc-stage-index-manual-release",
  "status_message": "",
  "status_reason": "Succeeded"
}
```